integration-tests/src/test/java/com/datastax/oss/driver/api/core/metadata/ByteOrderedTokenIT.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2017-2017 DataStax Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.driver.api.core.metadata;
+
+import com.datastax.oss.driver.api.core.Cluster;
+import com.datastax.oss.driver.api.core.session.Session;
+import com.datastax.oss.driver.api.testinfra.ccm.CustomCcmRule;
+import com.datastax.oss.driver.api.testinfra.cluster.ClusterRule;
+import com.datastax.oss.driver.categories.IsolatedTests;
+import com.datastax.oss.driver.internal.core.metadata.token.ByteOrderedToken;
+import com.datastax.oss.driver.internal.core.metadata.token.RandomToken;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.experimental.categories.Category;
+
+@Category(IsolatedTests.class)
```

Comment:
I don't think these need to be isolated (could just be long), unless i'm missing something,  however in hindsight the difference between long and isolated isn't really that much, maybe we should consider joining the two.

Reply:
They require a specific configuration of the CCM cluster (3 nodes, custom partitioner), doesn't that require isolation?

Reply:
the `IsolatedTests` category was made for tests that need their own JVM (for setting up system properties and such for SSL among other things).   The `CustomCcmRule` is set up such that there can only be one going at a given time (and they are shut down) so you can have multiple tests using `CustomCcmRule` in the same JVM.

However, the long tests by their nature are...long, and running each test class in its own JVM doesn't create all too much overhead, so we could consider merging those two categories if that makes sense?